build/tasks/test-on-node.js

```diff
@@ -5,63 +5,24 @@ module.exports = function( grunt ) {
 	grunt.registerMultiTask( "test-on-node", async function() {
 		var done = this.async();
 
-		const stats = [];
+		const totals = { files: 0, failed: 0 };
 		for ( const file of this.data ) {
-			const stat = await new Promise( resolve => runQUnit( file, resolve ) );
-			stats.push( stat );
+			totals.failed += await new Promise( resolve => runQUnit( file, resolve ) );
+			totals.files++;
 		}
 
-		var totals = stats.reduce( function( totals, stats ) {
-			totals.passed += stats.passed;
-			totals.failed += stats.failed;
-			totals.skipped += stats.skipped;
-			totals.todo += stats.todo;
-			totals.passedAssertions += stats.passedAssertions;
-			totals.failedAssertions += stats.failedAssertions;
-			totals.runtime += stats.runtime;
-			return totals;
-		}, {
-			passed: 0,
-			failed: 0,
-			skipped: 0,
-			todo: 0,
-			runtime: 0,
-			passedAssertions: 0,
-			failedAssertions: 0
-		} );
-
-		grunt.log.writeln( "-----" );
-		grunt.log.ok( constructMessage( totals ) );
+		if ( totals.failed ) {
+			grunt.log.error( `Ran ${totals.files} test files, ${totals.failed} failed` );
```

Comment:
I always have to double-check: is "failed" the number of files failed, or the number of assertions failed across all tests/files?
If it's the latter, maybe rewording to "Ran X test files, Y assertions failed". Otherwise it reads fine, and it just my own hesitation.

Reply:
Neither assertions nor files, it's the number of tests.

Updated to:
```js
Ran ${totals.files} files, ${totals.failed} failed tests
```